TextToSpeech/test/TTSThunderAPITest.cpp

```diff
@@ -263,7 +263,7 @@ int main(int argc, char *argv[]) {
                         &Handlers::onSpeechCancelledHandler) == Core::ERROR_NONE) {
                 LOGERR("Failed to Subscribe notification handler : onspeechcancelledHandler");
```

Comment:
This could be removed until we support queuing mechanism in TTS

Reply:
Yes. I have removed the cancel event subscription and handler.